docs/Common/Structs/index.md

```diff
@@ -411,3 +411,10 @@
 |imageTypeSupported|Common.FileType|false|array: true<br>minsize: 1<br>maxsize: 100||
 |imageResolution|Common.ImageResolution|false|||
 
+### ExternalConsentStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|entityType|Integer|true|minvalue: 0<br>maxvalue: 128|The entityType which status is informed by "status" param.|
+|entityID|Integer|true|minvalue: 0<br>maxvalue: 128|The corresponding ID of entityType which status is informed by "status" param.|
+|status|[Common.EntityStatus](https://github.com/KhrystynaDubovyk/sdl_hmi_integration_guidelines/blob/user_consent_in_terms_of_policies/docs/Common/Enums/index.md#entitystatus)|true|-|Status of External User Consent Settings entity: "ON" or "OFF"|
```

Comment:
Should be relative links, linking to GitHub is not a good idea on the actual dev portal, change to `../common/enums/#entitystatus`